admin/app.js

```diff
@@ -72,7 +71,7 @@ const mongoStoreOptions = {
 }
 const sessionOptions = {
   name: 'staff-app.sid',
-  secret: process.env.NODE_ENV === 'production' ? process.env.SESSION_SECRET : 'anti tamper for dev',
+  secret: process.env.NODE_ENV === 'production' ? config.SESSION_SECRET : 'anti tamper for dev',
```

Comment:
It would make sense to ensure that SESSION SECRET was populated earlier in the code, e.g. during the config initialisation.